carbon-design-system/carbon
# feat(multiselect): add onMenuChange event

Closes #7287

Adds onMenuChange event to multi select

#### Changelog

**New**

- onMenuChange event on multiselect

**Changed**

**Removed**


#### Testing / Reviewing

- Start storybook
- Open multiselect story
- Open the multiselect menu
- See `onMenuChange: (1) [true]`
- Close the multiselect menu
- See `onMenuChange: (1) [false]`
